addons/web_editor/static/src/scss/wysiwyg.scss

```
@@ -648,7 +651,11 @@ img.o_we_selected_image {
     // Prevent the edited link from being deselected when clicking between
     // buttons in the popover
     user-select: none;
-
+    box-shadow: $box-shadow;
+    border-color: rgba(0,0,0,0.025);
+    .popover-arrow::before{
```

Missing empty line before the selector and an empty space before the bracket ;)
```suggestion

    .popover-arrow::before {
```